Lib/test/test_asyncore.py

```diff
@@ -499,6 +499,12 @@ def handle_connect(self):
         pass
 
 
+class SendHandler(BaseTestHandler):
+    def __init__(self, conn):
+        BaseTestHandler.__init__(self, conn)
+        self.send(b'x' * 1024)
+
+
 class BaseTestAPI:
```

Comment:
It's not necessary but this was the `TestHandler` before defined inside `test_handle_read`, I pulled it out and renamed it to use it in this new test as well.